kernel/src/ds/bpf/test/hashmap_test.c

```diff
@@ -20,30 +20,30 @@ void simpleFree(void * val) {
 }
 
 TEST_CREATE(test_create_hm, {
-    HashMap * hm = hashmap__new(int_hash_fn, int_compare_fn, fakeFree, fakeFree, NULL);
-
-    int key = 5;
-    int data = 42;
-
-    hashmap__add(hm, &key, &data);
-
-    ASSERT(hashmap__find(hm, &key, NULL));
-
-    hashmap__free(hm);
+//    HashMap * hm = hashmap__new(int_hash_fn, int_compare_fn, fakeFree, fakeFree, NULL);
+//
```

Comment:
These tests should probably be uncommented, or fixed if they are  faulty.

Reply:
fixed